back/app/models/concerns/user_roles.rb

```diff
@@ -62,6 +62,11 @@ def _roles_json_schema
       where.not(id: project_folder_moderator(*project_folder_ids))
     }
 
+    scope :project_reviewers, lambda { |project_reviewer = true|
```

Comment:
Not sure I fully get this 🤔 Could you explain a bit how it works?

Reply:
It's to allow us to write something like:
```ruby
User.project_reviewers # => only project reviewers
User.project_reviewers(true) # => only project reviewers
User.project_reviewers(false) # => users that are not project reviewers
```

It's an (better I think) alternative to what we did for other scopes such as:
```ruby
User.admin
User.not_admin
```